cob_hardware_config/xacrodisplay.launch

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
+  <arg name="gui" default="false" />
+  <arg name="rvizconfig" default="$(find urdf_tutorial)/urdf.rviz" />
```

Comment:
the `cob_hardware_config/robots/$(arg robot)/$(arg robot).rviz` includes the cameras, the sensors data, map... all of this is not needed for the purpose of the file. I propose copy the file to `cob_hardware_config/robots/common/display_robot.rviz`, is this ok?